RecoBTag/Configuration/python/RecoBTag_cff.py

```diff
@@ -35,7 +36,10 @@
     softPFElectronBJetTags,
 
     # overall combined taggers
-    combinedMVAV2BJetTags
+    combinedMVAV2BJetTags,
+    
+    # pixel cluster
+    pixelClusterTagInfos,
```

Comment:
BTW, what is the meaning of "legacy" here and why is newly added ```pixelClusterTagInfos``` expected to belong to it?

Reply:
The legacy b-tagging sequence has been there a long time before this PR, and my understanding (BTV experts may want to comment) is that the difference with the standard sequence is the use of PF-based taggers vs non-PF. PixelClusters do not rely on PF but on low level information, so it should be the same for both PF and non-PF.